src/views/studio/SelectMembershipView/Steps/TermsStep.tsx

```diff
@@ -0,0 +1,127 @@
+import { absoluteRoutes } from '@/config/routes'
+import { Checkbox } from '@/shared/components'
+import Text from '@/shared/components/Text'
+import { transitions } from '@/shared/theme'
+import React, { useEffect, useRef, useState } from 'react'
+import { useNavigate } from 'react-router'
+import { CSSTransition } from 'react-transition-group'
+import { StepTitle, StepWrapper } from './Steps.style'
+import {
+  TermsBox,
+  TextWrapper,
+  TermsParagraph,
+  TermsOverlay,
+  ScrollButton,
+  TermsForm,
+  ContinueButton,
+} from './TermsStep.style'
+
+type TermsStepProps = {
+  onStepChange: (idx: number) => void
+}
+
+const TermsStep: React.FC<TermsStepProps> = ({ onStepChange }) => {
+  const navigate = useNavigate()
+  const [isCheckboxVisible, setIsCheckboxVisible] = useState(false)
+  const [isRead, setIsRead] = useState(false)
```

Comment:
What is `isRead` used for here? I don't see it having other purpose than `isCheckboxVisible`?